README.md

```diff
@@ -61,16 +61,26 @@ There five categories of metrics that Moneo monitors:
 
 Minimum Requirements
 -----
-
 - python >=3.7 installed
-- docker installed
-- ansible installed (python module)
+
 - OS Support:
-    - Ubuntu 18.04, 20.04
+    - Ubuntu 18.04, 20.04, 22.04
```

Comment:
Do we need to add mariner as well?

Reply:
I think that officially we have never tried Moneo on Mariner. So probably not have that here. Even if we use the Mariner docker for geneva agent